public/index.html

```diff
@@ -115,6 +116,17 @@ <h2>Resources</h2>
       <!-- the Leaflet map container -->
       <div id="map"></div>
 
+      <!--The search bar-->
+      <div id="search-bar-div">
```

Comment:
It would probably make sense to use `<form></form>` here, in terms of supporting a11y. Here are some useful links on how to make search bars accessible:
- http://web-accessibility.carnegiemuseums.org/code/search/
- https://www.a11ymatters.com/pattern/accessible-search/#building-the-search-component
- https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role
Note that both the `<label>` and `<button>` elements can be "visually hidden" so that they are available for users screen readers and other assistive technologies. [More on how to do that](https://webaim.org/techniques/css/invisiblecontent/)

Reply:
For the autocomplete, that will be a bit trickier to support a11y, but my intuition tells me that if you made the autocomplete results an unordered list with each list item containing a button, it would be a good start. [Here's an example (uses jQuery, but we could do it without jQuery)](https://haltersweb.github.io/Accessibility/autocomplete.html). And [here is a autocomplete component with a11y support on Github](https://github.com/alphagov/accessible-autocomplete) but uses Preact.

Reply:
Thanks so much for all these good suggestions Chris. Going to work on implementing these today or tomorrow.

Reply:
No problem!

Reply:
Regarding the autocomplete, I am lazy about keeping up with new html5 features so I didn't know autocomplete lists are a native feature of modern browsers using the `<datalist>` element. Native browser features are always better than hand-built things for a11y, so I'm going to change it to that, put in whatever aria needed to make it clear and add a polyfill for earlier browsers.